### PR TITLE
[postgres] Move pg to Debezium conversion

### PR DESCRIPTION
**Pull request by nathan-artie**

### CODE REVIEW

File: lib/postgres/debezium/payload.go [hunk 1]
```diff
@@ -48,18 +48,8 @@ func NewPayload(newArgs *NewArgs) (util.SchemaEventPayload, error) {
...
-		if err != nil {
-			return util.SchemaEventPayload{}, fmt.Errorf("failed to parseValue: %w", err)
-		}
-
-		parsedRowData[key] = val
-	}
-
 	payload := util.Payload{
-		After: parsedRowData,
+		After: newArgs.RowData,
```
Comment (nathan-artie) [inline lib/postgres/debezium/payload.go:52]: NewPayload is only called by `MessageBuilder.Next()`

File: lib/postgres/debezium/payload.go [hunk 2]
```diff
@@ -48,18 +48,8 @@ func NewPayload(newArgs *NewArgs) (util.SchemaEventPayload, error) {
 		}},
 	}
 
-	parsedRowData := make(map[string]interface{})
```
Comment (nathan-artie) [inline lib/postgres/debezium/payload.go:51]: This now happens here: https://github.com/artie-labs/reader/pull/128/files#diff-52af9dd77dd1a5a17331ffdb4d2a378ac7da5d124965ff23f1810fb15eb2d3d5R58

File: lib/postgres/message.go [hunk 3]
```diff
@@ -68,3 +74,21 @@ func (m *MessageBuilder) Next() ([]lib.RawMessage, error) {
 	}
 	return result, nil
 }
+
+func (m *MessageBuilder) ConvertRowToDebezium(row map[string]interface{}) (map[string]interface{}, error) {
```
Comment (Tang8330) [inline lib/postgres/message.go:78]: Lowercase?

**Review by Tang8330 — APPROVED** [review]

File: lib/postgres/message.go (hunk 3 above)
Comment (nathan-artie) [reply lib/postgres/message.go]: Good catch.

**Pull request merged**